src/app/shared/form/dynamic-fields/string.util.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Replaces all occurrences of a search value with a replace value in a given string.
+ *
+ * @param string - The string to perform the replacement on.
+ * @param searchValue - The value to search for in the string.
+ * @param replaceValue - The value to replace the search value with.
+ * @returns The modified string with all occurrences of the search value replaced.
+*/
+export function replaceAll(string: string, searchValue: string, replaceValue: string): string {
+    let temp = string.split(searchValue);
+    return temp.join(replaceValue);
+}
```

Comment:
"String.portotype.replaceAll()" is available since the "es2021" version of the javascript engine, however, dspace-angular uses an older version so "replaceAll()" is not available. 

See: https://ui.dev/es2021